examples/typescript/index.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"index.js","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":";AAAA,MAAM,OAAO,GAAG,OAAO,CAAC,SAAS,CAAC,CAAC;AAEnC,MAAM,GAAG,GAAkC,MAAM,CAAC,OAAO,GAAG,OAAO,EAAE,CAAC;AAEtE,GAAG,CAAC,GAAG,CAAC,GAAG,EAAE,UAAS,OAAO,EAAE,QAAQ;IACnC,OAAO,CAAC,GAAG,CAAC,OAAO,CAAC,GAAG,CAAC,CAAC;IAEzB,QAAQ,CAAC,IAAI,CAAC,aAAa,CAAC,CAAC;AACjC,CAAC,CAAC,CAAC;AAEH,GAAG,CAAC,MAAM,CAAC,IAAI,CAAC,CAAC;AAEjB,OAAO,CAAC,GAAG,CAAC,8BAA8B,CAAC,CAAC","sourcesContent":["const express = require(\"express\");\n\nconst app: import(\"express\").Application = module.exports = express();\n\napp.get('/', function(request, response) {\n    console.log(request.url);\n\n    response.send(\"Hello World\");\n});\n\napp.listen(3000);\n\nconsole.log('Express started on port 3000');\n"]}
```

Comment:
I believe this file should not be released, is the build result? same for the file tsconfig.build.info

Reply:
100% - I'll clean up